### PR TITLE
Fix/@typescript-eslint/prefer-optional-chain

### PR DESCRIPTION
**Pull request by dw-0**
depends on #3982

### CODE REVIEW

File: apps/api/src/app/user/user.service.ts
Comment (dtslvr) [inline apps/api/src/app/user/user.service.ts:1]: 😮

File: apps/api/src/services/market-data/market-data.service.ts
Comment (dtslvr) [inline apps/api/src/services/market-data/market-data.service.ts:1]: 😮

File: apps/api/src/services/queues/data-gathering/data-gathering.processor.ts
Comment (dtslvr) [inline apps/api/src/services/queues/data-gathering/data-gathering.processor.ts:1]: 😮

File: apps/api/src/services/queues/portfolio-snapshot/portfolio-snapshot.processor.ts
Comment (dtslvr) [inline apps/api/src/services/queues/portfolio-snapshot/portfolio-snapshot.processor.ts:1]: 😮

File: apps/client/src/app/components/admin-market-data/admin-market-data.component.ts
Comment (dtslvr) [inline apps/client/src/app/components/admin-market-data/admin-market-data.component.ts:1]: 😮

**Review by dtslvr — CHANGES_REQUESTED** [review]

**Review by dtslvr — APPROVED** [review]
Thanks, @dw-0! 😎

**Comment by dw-0** [comment]
No idea why those import shenanigans appeared again 😐 what a pita 😅

**Pull request merged**